playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -46,12 +46,16 @@
 
   # Replace dc/docker-registry certificate secret contents if set.
   - block:
+    - name: Load lib_openshift modules
+      include_role:
+      name: lib_openshift
+
     - name: Retrieve registry service IP
-      command: >
-        {{ openshift.common.client_binary }} get service docker-registry
-        -o jsonpath='{.spec.clusterIP}'
-        --config={{ mktemp.stdout }}/admin.kubeconfig
-        -n default
+      oc_service:
+        namespace: default
+        name: docker-registry
+        state: list
+        kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
```

Comment:
I do not believe that we allow people to override openshift.common.config_base. The only case where config_base would not be '/etc/origin' is for old installs (3.0, maybe 3.1), but during upgrade we symlink /etc/origin to /etc/openshift. Since we wouldn't support the older versions for changes that we land in master, I do not see why we couldn't make the assumption now.